lib/resource.nunjucks

```diff
@@ -138,20 +138,26 @@
                             {% for key, item in b.formParameters %}
                               {% include "./item.nunjucks" %}
                             {% endfor %}
+
+                            {% if b.properties %}
+                              <ul>
+                                {% for key, item in b.properties %}
+                                  {% include "./item.nunjucks" %}
+                                {% endfor %}
+                              </ul>
+                            {% endif %}
+
                           </ul>
                         {% endif %}
 
                         {% if b.type and not b.schema %}
-                          <p><strong>Type</strong>:</p>
-                          {% if b.properties %}
-                            <ul>
-                              {% for key, item in b.properties %}
-                                {% include "./item.nunjucks" %}
+                          <p><strong>Type:</strong></p>
+                            <div class='list-group'>
+                              {% for typeName in b.type %}
+                                {% include "./type.nunjucks" %}
```

Comment:
I tried to add and check the support for URI parameters and query parameters. The type.nunjucks still uses the item.nunjucks template. The type.nunjucks is only used for 'object' types that are only supported by 1.0. I added a check to support the basic types of 0.8 might the type template be used for a 0.8 file.
When I try to run the 0.8 example script I get parsererrors so I'm not sure if I'm able to check BC with 0.8.